<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OptPlatform</class>
 <widget class="QMainWindow" name="OptPlatform">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1366</width>
    <height>797</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>30</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>OptPlatform</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_main">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QWidget" name="widget_main" native="true"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1366</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="action_add_new"/>
    <addaction name="action_open_file"/>
    <addaction name="action_open_proj"/>
    <addaction name="action"/>
    <addaction name="separator"/>
    <addaction name="action_save"/>
    <addaction name="action_save_as"/>
    <addaction name="separator"/>
    <addaction name="action_close"/>
    <addaction name="separator"/>
    <addaction name="action_quit"/>
   </widget>
   <widget class="QMenu" name="menu_Edit">
    <property name="title">
     <string>编辑</string>
    </property>
    <addaction name="action_copy"/>
    <addaction name="action_paste"/>
    <addaction name="action_cut"/>
   </widget>
   <widget class="QMenu" name="menu_Settings">
    <property name="title">
     <string>设置</string>
    </property>
    <widget class="QMenu" name="menu_language">
     <property name="title">
      <string>语言</string>
     </property>
     <property name="icon">
      <iconset resource="../../../icons.qrc">
       <normaloff>:/pic/icons/语言翻译.png</normaloff>:/pic/icons/语言翻译.png</iconset>
     </property>
     <addaction name="separator"/>
     <addaction name="action_chinese"/>
     <addaction name="action_english"/>
    </widget>
    <widget class="QMenu" name="menu_theme">
     <property name="title">
      <string>颜色主题</string>
     </property>
     <property name="icon">
      <iconset resource="../../../icons.qrc">
       <normaloff>:/pic/icons/初始化.png</normaloff>:/pic/icons/初始化.png</iconset>
     </property>
     <addaction name="action_classical"/>
     <addaction name="action_dark"/>
    </widget>
    <addaction name="menu_language"/>
    <addaction name="menu_theme"/>
    <addaction name="action_solver"/>
   </widget>
   <widget class="QMenu" name="menu_Help">
    <property name="title">
     <string>帮助</string>
    </property>
    <addaction name="action_aboutSoftware"/>
   </widget>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>窗口</string>
    </property>
    <addaction name="action_model_tree"/>
    <addaction name="action_message_log"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_Settings"/>
   <addaction name="menu_Edit"/>
   <addaction name="menu"/>
   <addaction name="menu_Help"/>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>30</height>
    </size>
   </property>
   <property name="sizeGripEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>7</pointsize>
    </font>
   </property>
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="styleSheet">
    <string notr="true">
</string>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonIconOnly</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_add_new"/>
   <addaction name="action_open_file"/>
   <addaction name="action_open_proj"/>
   <addaction name="action_close_proj"/>
   <addaction name="action_save"/>
   <addaction name="action_save_as"/>
   <addaction name="action_select_mode"/>
   <addaction name="action_apply"/>
   <addaction name="action_last"/>
   <addaction name="action_next"/>
   <addaction name="action_stop"/>
   <addaction name="action_report"/>
  </widget>
  <widget class="QDockWidget" name="dockWidget_paramSetting">
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="allowedAreas">
    <set>Qt::RightDockWidgetArea</set>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_param"/>
  </widget>
  <widget class="QDockWidget" name="dockWidget_prompt">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_prompt"/>
  </widget>
  <action name="action_add_new">
   <property name="icon">
    <iconset resource="../../../icons.qrc">
     <normaloff>:/pic/icons/新建icon.png</normaloff>:/pic/icons/新建icon.png</iconset>
   </property>
   <property name="text">
    <string>新建</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="action_open_file">
   <property name="icon">
    <iconset resource="../../../icons.qrc">
     <normaloff>:/pic/icons/打开文件icon.png</normaloff>:/pic/icons/打开文件icon.png</iconset>
   </property>
   <property name="text">
    <string>打开文件</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="action_save">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../../../icons.qrc">
     <normaloff>:/pic/icons/保存icon.png</normaloff>:/pic/icons/保存icon.png</iconset>
   </property>
   <property name="text">
    <string>保存</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="action_save_as">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../../../icons.qrc">
     <normaloff>:/pic/icons/打印icon.png</normaloff>:/pic/icons/打印icon.png</iconset>
   </property>
   <property name="text">
    <string>另存为</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Alt+S</string>
   </property>
   <property name="visible">
    <bool>true</bool>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="action_close">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>关闭</string>
   </property>
  </action>
  <action name="action_quit">
   <property name="text">
    <string>退出</string>
   </property>
  </action>
  <action name="action_open_proj">
   <property name="icon">
    <iconset resource="../../../icons.qrc">
     <normaloff>:/pic/icons/打开项目.png</normaloff>:/pic/icons/打开项目.png</iconset>
   </property>
   <property name="text">
    <string>打开项目</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Alt+O</string>
   </property>
   <property name="visible">
    <bool>false</bool>
   </property>
  </action>
  <action name="action_copy">
   <property name="icon">
    <iconset resource="../../../icons.qrc">
     <normaloff>:/pic/icons/复制.png</normaloff>:/pic/icons/复制.png</iconset>
   </property>
   <property name="text">
    <string>复制</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="action_paste">
   <property name="icon">
    <iconset resource="../../../icons.qrc">
     <normaloff>:/pic/icons/粘贴.png</normaloff>:/pic/icons/粘贴.png</iconset>
   </property>
   <property name="text">
    <string>粘贴</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="action_cut">
   <property name="icon">
    <iconset resource="../../../icons.qrc">
     <normaloff>:/pic/icons/剪切.png</normaloff>:/pic/icons/剪切.png</iconset>
   </property>
   <property name="text">
    <string>剪切</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="action_aboutSoftware">
   <property name="icon">
    <iconset resource="../../../icons.qrc">
     <normaloff>:/pic/icons/帮助与说明.png</normaloff>:/pic/icons/帮助与说明.png</iconset>
   </property>
   <property name="text">
    <string>关于软件</string>
   </property>
   <property name="shortcut">
    <string>F12</string>
   </property>
  </action>
  <action name="action_model_tree">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../../../icons.qrc">
     <normaloff>:/pic/icons/树结构.png</normaloff>:/pic/icons/树结构.png</iconset>
   </property>
   <property name="text">
    <string>模型树</string>
   </property>
  </action>
  <action name="action_message_log">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../../../icons.qrc">
     <normaloff>:/pic/icons/审批日志.png</normaloff>:/pic/icons/审批日志.png</iconset>
   </property>
   <property name="text">
    <string>日志信息</string>
   </property>
  </action>
  <action name="action_select_mode">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../../../icons.qrc">
     <normaloff>:/pic/icons/模板选择icon.png</normaloff>:/pic/icons/模板选择icon.png</iconset>
   </property>
   <property name="text">
    <string>添加模块</string>
   </property>
   <property name="toolTip">
    <string>添加功能模块</string>
   </property>
  </action>
  <action name="action_close_proj">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../../../icons.qrc">
     <normaloff>:/pic/icons/关闭项目.png</normaloff>:/pic/icons/关闭项目.png</iconset>
   </property>
   <property name="text">
    <string>关闭项目</string>
   </property>
   <property name="visible">
    <bool>false</bool>
   </property>
  </action>
  <action name="action_close_module">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../../../icons.qrc">
     <normaloff>:/pic/icons/删除模块.png</normaloff>:/pic/icons/删除模块.png</iconset>
   </property>
   <property name="text">
    <string>删除模块</string>
   </property>
   <property name="toolTip">
    <string>删除模型树中选中的功能模块</string>
   </property>
  </action>
  <action name="action_apply">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../../../icons.qrc">
     <normaloff>:/pic/icons/开始计算icon.png</normaloff>:/pic/icons/开始计算icon.png</iconset>
   </property>
   <property name="text">
    <string>导入</string>
   </property>
  </action>
  <action name="action_last">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../../../icons.qrc">
     <normaloff>:/pic/icons/上一步icon.png</normaloff>:/pic/icons/上一步icon.png</iconset>
   </property>
   <property name="text">
    <string>上一步</string>
   </property>
  </action>
  <action name="action_next">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../../../icons.qrc">
     <normaloff>:/pic/icons/下一步icon.png</normaloff>:/pic/icons/下一步icon.png</iconset>
   </property>
   <property name="text">
    <string>下一步</string>
   </property>
  </action>
  <action name="action_report">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../../../icons.qrc">
     <normaloff>:/pic/icons/日志icon.png</normaloff>:/pic/icons/日志icon.png</iconset>
   </property>
   <property name="text">
    <string>报告</string>
   </property>
  </action>
  <action name="action_stop">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../../../icons.qrc">
     <normaloff>:/pic/icons/停止计算icon.png</normaloff>:/pic/icons/停止计算icon.png</iconset>
   </property>
   <property name="text">
    <string>停止</string>
   </property>
  </action>
  <action name="action_chinese">
   <property name="icon">
    <iconset resource="../../../icons.qrc">
     <normaloff>:/pic/icons/中国.png</normaloff>:/pic/icons/中国.png</iconset>
   </property>
   <property name="text">
    <string>中文</string>
   </property>
  </action>
  <action name="action_english">
   <property name="icon">
    <iconset resource="../../../icons.qrc">
     <normaloff>:/pic/icons/美国国旗.png</normaloff>:/pic/icons/美国国旗.png</iconset>
   </property>
   <property name="text">
    <string>英语</string>
   </property>
  </action>
  <action name="action_classical">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>经典</string>
   </property>
  </action>
  <action name="action_dark">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>深色</string>
   </property>
  </action>
  <action name="action">
   <property name="text">
    <string>打开最近</string>
   </property>
  </action>
  <action name="action_solver">
   <property name="icon">
    <iconset resource="../../../icons.qrc">
     <normaloff>:/pic/icons/模型设置.png</normaloff>:/pic/icons/模型设置.png</iconset>
   </property>
   <property name="text">
    <string>求解器</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../../../icons.qrc"/>
 </resources>
 <connections/>
</ui>
